docs/enhancement-proposals/off-ep-0005.md

```diff
@@ -0,0 +1,134 @@
+* We explicitly specify which self-consistent physical constants should be used to compute
+
+We leave these to future extensions:
+* We reserve the possibility of adding a `nonperiodic_potential` keyword at a future date should it become necessary to permit different choices for non-periodic systems
+* We reserve the possibility of adding specific keywords to specify reaction field potentials for `periodic_potential`
+
+## Usage and Impact
+
+Since most force fields use some flavor of PME for periodic systems and something similar to
+`nonperiodic_potential="Coulomb"` for non-periodic systems, the default attributes for this tag will
+likely be the most commonly-used. Splitting `method` out into explicit attributes to specify periodic,
+nonperiodic, and exception potential energy terms, however, makes it less
+ambiguous how electrostatics should be handled in each case and decouples the method used in each
+case.
+
+Users are recommended to consider upgrading from the default attribute values of 0.3 to 0.4 to avoid
+continuing to use this ambiguity. Implementations may wish to execute this up-conversion automatically (see below).
+
+If backward-compatibility is provided as specified below, users of old force fields will not need to update their force field definitions.
+
+## Backward compatibility
+
+To minimize pain in implementations, we can allow implementations continue to accept `method="PME"` as a synonym for `periodic_potential="Ewald3D-ConductingBoundary"` for a few releases,
+issuing a warning that the spec has changed and this behavior may be removed in a future release.
+
+Implementations may wish to add up-converters from old versions. An up-converter could convert the following tag header
+```
+<Electrostatics version="0.3" method="PME" scale12="0.0" scale13="0.0" scale14="0.833333" scale15="1.0"/>
+```
+to a less ambiguous header using version 0.4, which for this case could be
+```
+<Electrostatics version="0.4" periodic_potential="Ewald3D-ConductingBoundary" nonperiodic_potential="Coulomb" exception_potential="Coulomb" scale12="0.0" scale13="0.0" scale14="0.833333" scale15="1.0"/>
+```
+
+## Detailed description
+
+[CODATA 2018](https://physics.nist.gov/cuu/Constants/index.html) physical constants are used.
+Future OFF-EPs may migrate the specification of which self-consistent physical constants are used to a higher-level attribute.
+
+The `method` tag attribute is **removed** and replaced with `periodic_potential`, `nonperiodic_potential`, and `exception_potential`.
+
+The optional `cutoff` tag attribute is added to specify the cutoff used for `periodic_potential` if applicable, defaulting to `None`.
+No value of `cutoff` is allowed for Ewald methods---only finite-ranged methods.
+
+The optional `solvent_dielectric` tag attribute is added to specify the solvent dielectric used with finite-ranged potentials, defaulting to `78.5`.
```

Comment:
(not blocking) To other reviewers - I have no strong opinion about this default value, but you should be aware that `78.5` matches the default `solvent_dielectric` value in the [GBSA tag.](https://github.com/openforcefield/standards/blob/17e57c934e449b8ab79462bd2db245d73b26913b/docs/standards/smirnoff.md#gbsa)